doc/Configure.help

```diff
@@ -3240,3 +3240,15 @@ BSBPORT_MQTT_TOPIC
   Topic for publish BSBPORT_MQTT_TOPIC/SOURCE/MSG_ADDRESS/VALUE_TYPE payload is the value.
   Topic for querys BSBPORT_MQTT_TOPIC/query/DESTINATION/MSG_ADDRESS payload can by anything.
   Topic for setting values BSBPORT_MQTT_TOPIC/set/DESTINATION/MSG_ADDRESS/VALUE_TYPE payload is the value send.
+
+FAT date/time support
+SD_DATETIME_SUPPORT
```

Comment:
The option in config.in is named FAT_DATETIME_SUPPORT. Why is it named SD_DATETIME_SUPPORT in this file? Did you manually add this entry? Normaly menuconfig prepares the entry for you and includes the right names and dependings.